ts/theme/variables.ts

```diff
@@ -71,10 +71,15 @@ const customVariables = Object.assign(materialVariables, {
     return this.fontSizeBase * 1.75;
   },
   lineHeightBase: 24,
+
   get lineHeight1(): number {
     return this.lineHeightBase * 0.75;
   },
 
+  get lineHeight2(): number {
+    return this.lineHeightBase * (13 / 12);
```

Comment:
13 / 12?

Reply:
then `* 26/24` - also add the above reason as a comment so the next reader will know